services/caldar-api/src/controllers/customer-controller.ts

```diff
@@ -0,0 +1,159 @@
+import fs from "fs";
+import path from "path";
+import { Request, RequestHandler, Response } from "express";
+import { Customer } from "@caldar/types";
+import { CustomerType } from "@caldar/types";
+
+const customersDataFilePath = path.join(
+  process.cwd(),
+  "src",
+  "data",
+  "customers.json"
+);
+
+type queryType = {
+  type?: CustomerType;
+};
+
+export const getCustomers: RequestHandler = (req: Request, res: Response) => {
+  let response = getDataCustomers();
+
+  res.status(200).json({
+    status: "Ok",
+    content: response,
+  });
+};
+
+export const getCustomersByType: RequestHandler = (
+  req: Request,
+  res: Response
+) => {
+  const query: queryType = {};
+
+  query.type = req.query.type ? Number(req.query.type) : undefined;
+
+  let customers = getDataCustomers();
```

Comment:
Let's try to use `const` when a variable is not re-assigned
```suggestion
  const customers = getDataCustomers();
```